angular/src/lib/topbar/topbar.component.ts

```diff
@@ -18,14 +18,17 @@ export type TopbarColor = 'dark' | 'light' | 'blue';
     'attr.role': 'navigation',
     class: 'md-top-bar',
     '[class.md-top-bar--fixed]': 'fixed',
+    '[class.md-top-bar--absolute]': 'absolute',
     '[class.md-top-bar--blue]': 'color === "blue"',
     '[class.md-top-bar--dark]': 'color === "dark"',
     '[class.md-top-bar--light]': 'color === "light"',
   },
 })
 export class TopbarComponent {
-  /** @option Topbar header color | '' */
+  /** @prop Topbar header color | '' */
   @Input() public color: TopbarColor;
-  /** @option Determines if Topbar is fixed to top | false */
+  /** @prop Determines if Topbar is fixed to top | false */
   @Input() public fixed: boolean;
+  /** @prop Determines if Topbar is absolute to top in a container | false */
+  @Input() public absolute: boolean;
```

Comment:
Should we consider changing this to a new `position` prop with fixed or absolute values?

Reply:
I'm good with merging this and then discussing how we should move forward on all of the libraries.